speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/BrakeDecision.cpp

```diff
@@ -1,6 +1,6 @@
 #include "BrakeDecision.h"
 #include "Mediator.h"
-#define BRAKE_THRESHOLD 0.2f
+#define BRAKE_THRESHOLD 0.9f
```

Comment:
Isn't this too high to be able to test the icons? (for example the client blackbox is always around ~0.5)

Reply:
No, IMO the black box should indicate a full stop, which is above 0.9. Otherwise we can always change this constant in the future